src/store/actions/login.js

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+import { LOGIN } from './types';
+import { API_BASE_URL } from '../../settings';
+import { SUCCESS, FAILURE } from '../../constants';
+
+export const login = (status, payload) => ({
+  type: LOGIN[status],
+  payload,
+});
+
+export const loginAPI = user => async (dispatch) => {
```

Comment:
`loginAPI` isn't a clear description for a function. Function names should typically be verbs as they describe the logic that is contained within them. `loginAPI` however is a noun. 

Once you have the appropriate middleware, you actually shouldn't need this `loginAPI` function as it will be automatically handled.

I'll leave you to research the implementation of the middleware to leverage a more DRY structure.

Reply:
Yup. I will mark it as resolved as it will no longer be needed once I implement the middleware API on my next PR.